content/hacking-atom/sections/the-init-file.md

```diff
@@ -3,7 +3,7 @@ title: The Init File
 ---
 ### The Init File
 
-When Atom finishes loading, it will evaluate `init.coffee` in your `~/.atom` directory, giving you a chance to run CoffeeScript code to make customizations. Code in this file has full access to [Atom's API](https://atom.io/docs/api/latest). If customizations become extensive, consider creating a package, which we will cover in link:../package-word-count[package-word-count].
+When Atom finishes loading, it will evaluate `init.coffee` in your `~/.atom` directory, giving you a chance to run CoffeeScript code to make customizations. Code in this file has full access to [Atom's API](https://atom.io/docs/api/latest). If customizations become extensive, consider creating a package, which we will cover in [../package-word-count](Package: Word Count).
```

Comment:
It looks like you got the text and URL reversed:

https://help.github.com/articles/basic-writing-and-formatting-syntax/#links